Create toil-rnaseq-unc.sop.md

wshands opened the pull request:
This is a SOP that outlines the steps that need to be taken to run the RNA-seq UNC pipeline that are different from the steps that are used to run the RNA-seq CGL pipeline.

wshands committed:
Create toil-rnaseq-unc.sop.md

This is a SOP that outlines the steps that need to be taken to run the RNA-seq UNC pipeline that are different from the steps that are used to run the RNA-seq CGL pipeline.

wshands committed:
Update toil-rnaseq-unc.sop.md

added suggestions for updating the launch_unc_mesos.sh script

hannes-ucsc commented on SOPs/toil-rnaseq-unc.sop.md:
```diff
@@ -0,0 +1,90 @@
+`git clone toil-scripts -b releases/2.0.x`
+you'll also need to use `cgcloud ssh-cluster` to `pip uninstall toil` and reinstall `pip install toil==3.1.6` to all the nodes
+ 
+There is a convenience script for running on mesos, which you'll edit to change bucket other settings etc.  
```
Mark-up should be wrapped at 80 characters. While that wrapping does affect the rendered output, it makes  the source more readable.

hannes-ucsc commented on SOPs/toil-rnaseq-unc.sop.md:
```diff
@@ -0,0 +1,90 @@
+You will also need to have the master key file in the shared directory
+
+1) Setup CGCloud
+See the SOP https://github.com/BD2KGenomics/core-operations/blob/master/SOPs/toil-rnaseq.sop.md
```
A relative link would be better. If I were looking at this file in a branch, an absolute link would lead me to the file on master. And that's typically not what you'd want.

hannes-ucsc commented on SOPs/toil-rnaseq-unc.sop.md:
```diff
@@ -0,0 +1,90 @@
+2) Launch a cluster
+See the SOP https://github.com/BD2KGenomics/core-operations/blob/master/SOPs/toil-rnaseq.sop.md
+
+#instead of section 3 and 4 in the toil-rnaseq.sop.md use the following:
```
`# Instead …` is more readable and more grammatically correct.

hannes-ucsc replied:
I would refer to the other SOP as the `RNASeq SOP`. You could even use that as the title for the links above.

hannes-ucsc commented on SOPs/toil-rnaseq-unc.sop.md:
```diff
@@ -0,0 +1,90 @@
+
+#instead of section 3 and 4 in the toil-rnaseq.sop.md use the following:
+3) Install and configure the pipeline
+#ssh as administrator into the Toil leader (-a) so you can do sudo apt-get to install git
```
I would just say `… so you have sudo rights`

hannes-ucsc commented on SOPs/toil-rnaseq-unc.sop.md:
```diff
@@ -0,0 +1,90 @@
+exit
+
+#grow the cluster
+cgcloud grow-cluster --cluster-name example --spot-auto-zone --spot-bid 1.00 --instance-type c3.8xlarge --num-workers 22 toil
```
Do you want to say sth about the actual spot bid?

hannes-ucsc commented on SOPs/toil-rnaseq-unc.sop.md:
```diff
@@ -0,0 +1,90 @@
+#install Toil version 3.1.6 on all the workers
+cgcloud ssh-cluster --cluster-name example  --admin toil -o StrictHostKeyChecking=no sudo pip install toil==3.1.6
+
+#install unzip and samtools because these are not installed by default
```
Drop `because …`

hannes-ucsc commented on SOPs/toil-rnaseq-unc.sop.md:
```diff
@@ -0,0 +1,90 @@
+cgcloud ssh --cluster-name example toil-leader
+
+#clone the correct toil scripts repo
+git clone https://github.com/BD2KGenomics/toil-scripts -b releases/2.0.x
```
This should refer to a tag, not a branch.

kozbo commented:
Please complete!

Commit:
Fix comments

Add fixes to pull request based on https://github.com/BD2KGenomics/core-operations/pull/91
